doc/runtimes/hugo.md

```diff
@@ -16,6 +16,8 @@ The Hugo toolset is run when one of following conditions met:
 2. Recursively look up config directory: `config/**/*.yaml`, `config/**/*.toml`,
    `config/**/*.yml` or `config/**/*.json` configuration files in sub-directories.
 
+**Note**: Starting from [Hugo version `0.110.0`](https://gohugo.io/getting-started/configuration/#hugotoml-vs-configtoml), configuration file name has changed from `config.toml` etc. to `hugo.toml` etc. Please note the old naming convention is still supported.
```

Comment:
We could probably scrap this line in favor of updating the (1.) bullet above to mention that we look for the `hugo.*` equivalent files.